lib/hanami/events/adapter/redis.rb

```diff
@@ -4,30 +4,31 @@ module Hanami
   module Events
     class Adapter
       class Redis
-        EVENTS_NAME = 'hanami_event'
+        CHANNEL = 'hanami_events'
 
-        attr_reader :listeners
+        attr_reader :subscribers
 
-        def initialize(redis:)
-          @redis = redis
-          @listeners = []
-          @uniq_thread = false
+        def initialize(params)
+          @redis = params[:redis]
+          @subscribers = []
         end
 
-        def announce(event_name, payload)
-          @redis.publish(EVENTS_NAME, payload.to_json)
+        def broadcast(event_name, payload)
+          payload[:event_name] = event_name
+
+          @redis.publish(CHANNEL, payload.to_json)
```

Comment:
so, we can change this lines to `@redis.publish(CHANNEL, { event_name: event_name, **payload }.to_json)`